### ISSUE TITLE
AggregationsMultiTermLookup type is incorrect

### ISSUE THREAD

**Issue by stephmilovic** [elastic/elasticsearch#95628]
### Elasticsearch Version

8.8.0

### Java Version

N/A

### OS Version

N/A

### Problem Description

The multi-terms aggregation supports the `missing` field in the `terms` object:
```
  ...
  aggs: {
    groupByFields: {
      multi_terms: {
        terms: [
          {
            field: 'host.name',
            missing: '-',
          },
          {
            field: 'user.name',
            missing: '-',
          },
        ]
      },
      ...
```

However, the type is not supported:
```
Type '{ field: string; missing: string; }' is not assignable to type 'AggregationsMultiTermLookup'.   Object literal may only specify known properties, and 'missing' does not exist in type 'AggregationsMultiTermLookup
```

In elasticsearch-js is a term called `AggregationsMultiTermLookup`. It should have a property of `missing?: string | number`. Interesting thing is this field appeared in 8.7 and disappeared in 8.8:

- [8.6: no](https://github.com/elastic/elasticsearch-js/blob/8.6/src/api/typesWithBodyKey.ts#L3379)
- [8.7: yes](https://github.com/elastic/elasticsearch-js/blob/8.7/src/api/typesWithBodyKey.ts#L3457)
- [8.8: no](https://github.com/elastic/elasticsearch-js/blob/8.8/src/api/typesWithBodyKey.ts#L3379)

Can we please get it put back in for 8.8+?

### Steps to Reproduce

Look at the code, realize it the field is absent in 8.8:

- [8.6: no](https://github.com/elastic/elasticsearch-js/blob/8.6/src/api/typesWithBodyKey.ts#L3379)
- [8.7: yes](https://github.com/elastic/elasticsearch-js/blob/8.7/src/api/typesWithBodyKey.ts#L3457)
- [8.8: no](https://github.com/elastic/elasticsearch-js/blob/8.8/src/api/typesWithBodyKey.ts#L3379)

**Comment by JoshMock** [comment]
> Look at the code, realize it the field is absent in 8.8:
> 
>     * [8.6: no](https://github.com/elastic/elasticsearch-js/blob/8.6/src/api/typesWithBodyKey.ts?rgh-link-date=2023-04-27T14%3A13%3A50Z#L3379)
> 
>     * [8.7: yes](https://github.com/elastic/elasticsearch-js/blob/8.7/src/api/typesWithBodyKey.ts?rgh-link-date=2023-04-27T14%3A13%3A50Z#L3457)
> 
>     * [8.8: no](https://github.com/elastic/elasticsearch-js/blob/8.8/src/api/typesWithBodyKey.ts?rgh-link-date=2023-04-27T14%3A13%3A50Z#L3379)

This is because I haven't generated an 8.8 client yet. The branch exists because automation set it up this week, but it branched off of `main`, which hasn't built off of the `main` branch of the spec in quite a while. I'm fixing a couple bugs blocking that build, and then will push up changes to 8.8 soon. I'll let you know here when that's done.

**Comment by JoshMock** [comment]
@stephmilovic @rudolf I just published `elasticsearch-canary@8.8.0-canary.1` to npm a few minutes ago. Feel free to update and let me know if you run into any issues.

**Comment by stephmilovic** [comment]
Woohoo, appreciate it @JoshMock. Thank you!
